test/game_test.rb

```diff
@@ -20,6 +20,7 @@ def setup
                 :venue => "Heaven",
                 :venue_link => "venue/link"})
 
+    Game.from_csv('./test/fixtures/short_games.csv')
     Game.from_csv('./test/fixtures/games_20.csv')
```

Comment:
I'm curious about how this test is working with both fixture files being called.